.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended"
+  ],
+  "ignorePatterns": ["**/node_modules/", "out/**", "test/"],
+  "parser": "@typescript-eslint/parser",
```

Comment:
Is there any part of this configuration that will cause issues for people who don't use typescript/don't have typescript tooling installed?

My read is that it won't but I wanted to double check.

Reply:
`njs-type` is a TypeScript project.  the s3-gw is a JS project, and it seems it doesn't cause any issues atm